CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [unreleased]
+
+* Updated `glob` to the latest major version `^10.3.1` and refactored code to use promises instead of callbacks. 
```

Comment:
```suggestion
* Updated `glob` to the latest major version `^10.3.1` and refactored code. 
```
Changelog should write a short 'what to do'.
You don't have to write down 'how to do' in detail.